wireshark/pulsarDissector.cc

```diff
@@ -16,15 +16,13 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-#include <config.h>
 #include <epan/column-utils.h>
 #include <epan/dissectors/packet-tcp.h>
-#include <epan/expert.h>
 #include <epan/packet.h>
-#include <epan/prefs.h>
 #include <epan/proto.h>
 #include <epan/value_string.h>
 #include <glib.h>
+#include <ws_version.h>
```

Comment:
This header does not work for the `wireshark-dev` (3.2.3) on Ubuntu. It seems to be caused by the breaking change of the wireshark.

Anyway, currently we can focus on the wireshark 4.x support. So this comment is not a blocker.

Reply:
Yes, we can fix it for old version support later if necessary.